### PR TITLE
[reservations] Fix validation errors as follow up to #10297

### PR DESCRIPTION
**Pull request by aditink**
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i>

### Contribution checklist:
- [ ] I commit to follow the [Breaking Change Policy](http://aka.ms/bcforapi) of “no breaking changes
- [ ] I have reviewed the [documentation](https://aka.ms/ameonboard) for the workflow.
- [ ] [Validation tools](https://aka.ms/swaggertools) were run on swagger spec(s) and errors have all been fixed in this PR. [How to fix?](https://aka.ms/ci-fix)

If any further question about AME onboarding or validation tools, please view the [FAQ](https://aka.ms/faqinprreview).

### ARM API Review Checklist
- [ ] Ensure to check this box if one of the following scenarios meet updates in the PR, so that label “WaitForARMFeedback” will be added automatically to involve ARM API Review. Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs, all “removals” and “adding a new property” no more require ARM API review.
  - Adding new API(s)
  - Adding a new API version
  - Adding a new service

- [ ] If you are blocked on ARM review and want to get the PR merged with urgency, please get the ARM oncall for reviews (*RP Manifest Approvers* team under <ins>Azure Resource Manager service</ins>) from IcM and reach out to them. 

### Breaking Change Review Checklist 
If there are following updates in the PR, ensure to request an approval from API Review Board as defined in the [Breaking Change Policy](http://aka.ms/bcforapi). 

- [ ] Removing API(s) in stable version
- [ ] Removing properties in stable version
- [ ] Removing API version(s) in stable version
- [x] Updating API in stable version with Breaking Change Validation errors
- [ ] Updating API(s) in preview over 1 year

Please follow the link to find more details on [PR review process](https://aka.ms/SwaggerPRReview).

### CODE REVIEW

**Comment by raych1** [comment]
@jianyexi , could you give suggestions on the fix on lintDiff error and breaking change error?

**Comment by aditink** [comment]
Would greatly appreciate suggestions, as I am leaving at the end of the week and would like to avoid having to transition the work to someone else.

File: specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json [hunk 1]
```diff
@@ -1647,11 +1725,9 @@
       "x-ms-parameter-location": "method",
       "in": "body",
       "required": true,
+      "type": "object",
```
Comment (jianyexi) [inline specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json:1728]: The type should be removed

Comment (aditink) [reply specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/reservations.json]: removed, thanks!

**Comment by markcowl** [comment]
@aditink  This change requires a breaking change review with the Azure API Review Board

**Comment by aditink** [comment]
I no longer work at Microsoft. Someone else will need to pick up the PR.

**Comment by raych1** [comment]
The purpose is to fix the examples and should seek an approach without introduce breaking change. I close this PR since there is no bandwidth now to do this work from RP's owner perspective.

**Pull request closed without merging**